src/forms/renderers/MultiLineSecret.tsx

```diff
@@ -85,25 +85,6 @@ const MultiLineSecretRenderer = (props: any) => {
                 open={isUploadOpen}
                 onSave={submitFile}
                 filesLimit={1}
-                // List taken from -> https://pki-tutorial.readthedocs.io/en/latest/mime.html
-                acceptedFiles={[
-                    '.pem',
-                    '.p8',
-                    '.p10',
-                    '.p12',
-                    '.key',
-                    '.csr',
-                    '.cer',
-                    '.cr1',
-                    '.crt',
-                    '.p7c',
-                    '.p7b',
-                    '.p7r',
-                    '.der',
-                    '.pfx',
-                    '.spc',
-                    '',
-                ]}
```

Comment:
@mdibaiee is of the opinion these are not really needed. So removing for now. I mainly added them to just be super extra safe but they mainly just get in the way.